packages/web/components/MobileFooter.tsx

```diff
@@ -166,9 +167,9 @@ export const MobileFooter: React.FC = () => {
         <Stack
           position="fixed"
           left="0"
-          top="0"
+          bottom="5rem"
           width="100vw"
-          height="calc(100vh - 5rem)"
+          height="min(100vh, 36rem)"
```

Comment:
What's the intention behind these changes?

Reply:
This prevents the drawer from expanding full-screen on e.g. tablets where it's way too big. When the view height is < 36rem.